src/vs/workbench/browser/parts/compositeBarActions.ts

```diff
@@ -158,7 +158,15 @@ export class ActivityActionViewItem extends BaseActionViewItem {
 		if (this.label) {
 			if (this.options.icon) {
 				const foreground = this._action.checked ? colors.activeBackgroundColor || colors.activeForegroundColor : colors.inactiveBackgroundColor || colors.inactiveForegroundColor;
-				this.label.style.backgroundColor = foreground ? foreground.toString() : '';
+				const isExtension = this.label.className.indexOf('extensionViewlet-') >= 0;
+				const isRemoteExplorer = this.label.className.indexOf('remote') >= 0;
```

Comment:
@misolori  But why do you need to treat isRemoteExplorer differently?
Why doesn't that get caught as part of isExtensions?

@sandy081 would know more, however he is on vacation
@alexr00 might know more

Reply:
It’s because the remote explorer isn’t an extension contribution anymore and doesn’t follow the same naming pattern (posted in our slack channel about this).

Reply:
The remote explorer isn't in an extension anymore. It is now part of core, and then extension contribute to it.

Reply:
@misolori  If we go with this hack approach please be sure to create a debt item for you and for @sandy081 which you can tackle once he is back.
Probably the view should just have a specific class name to differntiate it.

Thanks

Reply:
I updated the pr so that the remote explorer uses the icon font, so the `isRemote` lines are no longer needed.